micrometer-core/src/test/java/io/micrometer/core/instrument/binder/logging/LogbackMetricsTest.java

```diff
@@ -86,6 +93,22 @@ void removeFilterFromLoggerContextOnClose() {
         assertThat(loggerContext.getTurboFilterList()).isEmpty();
     }
 
+    @Issue("#1282")
+    @Test
+    void compositeMeterRegistryShouldNotIncrementCounter() {
+        SimpleMeterRegistry simpleMeterRegistry = new SimpleMeterRegistry();
+        CompositeMeterRegistry compositeMeterRegistry = new CompositeMeterRegistry();
+        compositeMeterRegistry.add(simpleMeterRegistry);
+
+        new LogbackMetrics().bindTo(compositeMeterRegistry);
+        new LogbackMetrics().bindTo(simpleMeterRegistry);
```

Comment:
Why do we bind to each registry? It seems like it should just be binding to the composite registry.

Reply:
@checketts Creating `LogbackMetrics` twice is a mistake but applying `MeterBinder`s to all meter registries including composite meter registries is the current implementation in Spring Boot.

I think you're right if there's composite meter registry but if there isn't, we need to bind meters to all meter registries.

See https://github.com/micrometer-metrics/micrometer/issues/1282#issuecomment-472375431